### ISSUE TITLE
Ask CRAN to identify and quantify the resources it needs.

### ISSUE THREAD

**Issue by wlandau** [cranchange/cran_change.org#7]
Regarding this point:

> We appreciate a lot of reform is being sought from what is a famously small team of administrators. However, being under-resourced does not mitigate the shortcomings of CRAN's current practices, rather it is an argument in itself that reform is needed so that CRAN may attract the resources to perform its valuable role sustainably.

I agree, and I suggest we ask CRAN to identify and quantify the resources it needs to operate properly. I think these include:

* Number of administrators.
* Number of hours per week per administrator.
* Monetary compensation for administrators' time.
* Cost to maintain the website http://cran.r-project.org
* Cost to maintain CRAN servers, including but by no means limited to:
    * Hardware for repairs. 
    * Proprietary software fees, e.g. Windows/Mac OS/Solaris.

That way the community could pitch in to give them what they need. I bet the R Consortium could help with compensation.

**Comment by wlandau** [comment]
Related: speaking generally, it might be nice to acknowledge the hard work CRAN administrators do and offer community support. There may be more ways we can help them implement the proposed reforms.

**Comment by jonocarroll** [comment]
Related, I'd suggest rewording the following point
> Publish an account of resources CRAN receives, detailing how they are spent

as I believe it detracts from the primary message of the letter (verging on an implication of impropriety). I don't necessarily believe that package authors are entitled to an audit of CRAN's finances simply due to our (strong) disagreement with their practices. The R foundation may wish to pursue that, but that's between those parties.

I'd certainly like to hear of where their finances offer opportunities for the community to help remedy some of the shortcomings, but I don't think the 'show your books' approach is helpful towards the message here.

**Comment by MilesMcBain** [comment]
Ah okay thanks @jonocarroll, I didn't mean it to be read this way. It was more about understanding what resources it takes to run CRAN along the lines of what @wlandau outlined.

**Comment by MilesMcBain** [comment]
@wlandau I really appreciate your last point there. We'll work that in.

**Comment by MilesMcBain** [comment]
does the above commit address this issue? @jonocarroll  @wlandau

**Comment by llrs** [comment]
With "resources" do you include software used to receive package, manage and check them, sync mirrors, etc... ? Or it is just to mean servers, people and time dedicated?

**Comment by wlandau** [comment]
I was thinking about things that would require time, money, and/or people, anything the community is capable of contributing. Transparency about the specs of CRAN platforms could help users avoid surprises when preparing submissions, but that seems like a different issue.

@jonocarroll, I fully agree with https://github.com/MilesMcBain/cran_change.org/issues/7#issuecomment-842651680. Hopefully we can make it a "help us help you" message that does not look like we are trying to audit CRAN.

**Comment by jonocarroll** [comment]
@MilesMcBain I'd perhaps suggest something less widely demanding. e.g.

> Publish a breakdown of the shortfall in resources CRAN needs to function properly towards which the community could contribute. The R community greatly values the time and effort the CRAN team devotes towards maintaining this valuable service and would surely welcome any requests for contributions towards tooling which could lighten the burden of reviewing submissions.

**Comment by MilesMcBain** [comment]
I think the first sentence could work. The rest is kind of restating what I added in the Finally, ... para?

**Comment by MilesMcBain** [comment]
Ok I think I have a solution for this. I guess while I am curious, I don't really need an inventory of resources that they use, I only need assurances that they have them, and that they are not above asking where there is a shortfall, so:

> 11. Establish a process whereby CRAN notifies the R community of shortfalls in resources which could be met by community contributions

**Comment by wlandau** [comment]
Yeah, that sounds much better than what I originally suggested.
